__tests__/app.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require('supertest')
+const seed = require('../db/seeds/seed')
+const app = require('../app')
+const db = require('../db/connection')
+const testData = require('../db/data/test-data')
+
+
+beforeEach(()=>seed(testData))
+
+afterAll(()=>{
+    if(db.end) db.end();
+})
+
+describe('1. GET/api/topics', ()=>{
+    test('status 200, should respond with topics objects in array',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body.topics).toEqual(expect.any(Array));
```

Comment:
Testing that the array is at least greater than 0 is an assertion that should be added here as well to make sure an empty array does not pass this test.